repos/system_upgrade/el8toel9/actors/checkcustomnetworkscripts/actor.py

```diff
@@ -0,0 +1,21 @@
+import os
+import errno
+
+from leapp.actors import Actor
+from leapp.libraries.actor import customnetworkscripts
+from leapp.models import Report
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+
+class CheckCustomNetworkScripts(Actor):
+    """
+    Check the existence of custom network-scripts and warn user about possible
+    manual intervention requirements.
+    """
+
+    name = "custom_network_scripts_check"
```

Comment:
Regarding the issues, I suggest to be consistent in the name of the actor and strings used fot the class name and directory. E.g.

```suggestion
    name = "check_custom_network_scripts"
```

Reply:
Thanks for suggestion, I updated.